lib/components/dropdown-item.vue

```diff
@@ -1,31 +1,21 @@
 <template>
     <b-link class="dropdown-item"
             role="menuitem"
```

Comment:
Yeah, basically.  And buttons respect the disabled prop, so no need to do any special handling.